connector/src/main/java/org/jboss/as/connector/subsystems/resourceadapters/ResourceAdaptersRootResourceDefinition.java

```diff
@@ -18,6 +19,10 @@
  */
 public class ResourceAdaptersRootResourceDefinition extends SimpleResourceDefinition {
 
+    private static final String RESOURCE_ADAPTERS_SUBSYSTEM_CAPABILITY = "org.wildfly.resource-adapters";
```

Comment:
Shouldn't this be named RESOURCE_ADAPTERS_SUBSYSTEM_CAPABILITY_NAME and be shared in a "Capabilities.java " class instead of duplicating the constant on several places.

@see https://github.com/wildfly/wildfly/blob/main/messaging-activemq/subsystem/src/main/java/org/wildfly/extension/messaging/activemq/Capabilities.java for example